<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GesUser</class>
 <widget class="QDialog" name="GesUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Utilisateurs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/perso4.jpg</normaloff>:/icones/Icones/perso4.jpg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="agentEdit">
           <property name="placeholderText">
            <string>Agent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="profilComboBox">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Secrétariat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Caisse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chef Personel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DAF &amp; DG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admin</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Activer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Désactiver</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btActualiser">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_down_64.png</normaloff>:/icones/Icones/arrow_down_64.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="userTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btNouveau">
         <property name="text">
          <string>Nouveau</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btModifier">
         <property name="text">
          <string>Modifier</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/moduser.jpg</normaloff>:/icones/Icones/moduser.jpg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDesactiver">
         <property name="text">
          <string>Désactiver</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/desUser.jpg</normaloff>:/icones/Icones/desUser.jpg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btReactiver">
         <property name="text">
          <string>Reactiver</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/util1.jpg</normaloff>:/icones/Icones/util1.jpg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSupprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/suprUser.jpg</normaloff>:/icones/Icones/suprUser.jpg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>GesUser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
